src/FSharp.Build/Microsoft.FSharp.Targets

```diff
@@ -365,6 +365,7 @@ this file.
               PreferredUILang="$(PreferredUILang)"
               ProvideCommandLineArgs="$(ProvideCommandLineArgs)"
               PublicSign="$(PublicSign)"
+              RealSig="$(RealSig)"
```

Comment:
Outside of proto and plain dotnet build, we should now be fine with using `<RealSig>true</>` in this repo, or?

Reply:
Sigh!!!, vlad and some folk still like using building_using_dotnet, that build task doesn't have realsig yet.